src/utils/helpers.js

```diff
@@ -244,3 +244,11 @@ export function tokenInvalid({ storage, client_id, reauth }) {
 
   return false
 }
+
+export function isNode() {
+  return (
```

Comment:
`return process?.title === 'node'`
?

Reply:
@cjmarkham if you like. but i just stole it from the most popular npm lib for this so its apparently 'foolproof' 👀

Reply:
I'd stick with what's tested then :)